base/linalg/uniformscaling.jl

```diff
@@ -171,6 +172,14 @@ function isapprox(J1::UniformScaling{T}, J2::UniformScaling{S};
             rtol::Real=Base.rtoldefault(T,S), atol::Real=0, nans::Bool=false) where {T<:Number,S<:Number}
     isapprox(J1.λ, J2.λ, rtol=rtol, atol=atol, nans=nans)
 end
+function isapprox(J::UniformScaling,A::AbstractMatrix;
+                  rtol::Real=rtoldefault(promote_leaf_eltypes(A),eltype(J)),
+                  atol::Real=0, nans::Bool=false, norm::Function=vecnorm)
+    n = checksquare(A)
+    Jnorm = norm == vecnorm ? abs(J.λ)*sqrt(n) : norm(J)
```

Comment:
Committed. It might be pretty bad for sparse matrices though (e.g. it's conceivable that it would be used in a code that would only operate on very large sparse matrices and be prohibitively expensive), which is why I liked the previous option of just erroring out if the norm is not supported (in which case the user can implement it themselves) but admittedly that's a corner of a corner case.